content/kubermatic/main/architecture/concept/kkp-concepts/applications/default-applications-catalog/k8sgpt/_index.en.md

```diff
@@ -0,0 +1,36 @@
++++
+title = "K8sGPT Application"
+linkTitle = "K8sGPT"
+enterprise = true
+date = 2024-01-16T12:57:00+02:00
+weight = 11
+
++++
+
+# What is K8sGPT?
+K8sGPT gives Kubernetes SRE superpowers to everyone. 
```

Comment:
For me this sentence is a bit over the top :D I personally would remove it, but feel free to leave in if you like it.

Reply:
This one I pick from their official site.. So would keep it ;)

Reply:
okay 👍